sdk/synapse/synapse-artifacts/package.json

```diff
@@ -110,10 +111,6 @@
   "autoPublish": true,
   "//metadata": {
     "constantPaths": [
-      {
-        "path": "src/artifactsClientContext.ts",
-        "prefix": "packageDetails"
-      },
```

Comment:
don't think this should be removed. I remember it has something to do with the post release jobs.
 @praveenkuttappan  @joheredi Can you help confirm ?

Reply:
@joheredi Does that mean there is a codegen bug and it should be like:
`{
        "path": "src/artifactsClient.ts",
        "prefix": "packageDetails"
     }
`